BangazonAPI/Controllers/EmployeeController.cs

```diff
@@ -0,0 +1,317 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System.Data;
+using System.Data.SqlClient;
+using BangazonAPI.Models;
+using Microsoft.AspNetCore.Http;
+
+//This is the employee controller, which should have get all, one, create, and update functionality. Additionally, there is a request for delete which will ONLY run if you use the super secret code as a query in your request (PeteyDeletey=true). This is to prevent the test for this controller from flooding the database. Whether you fetch all employees or just one, you should see an employee with their department and current computer, if applicable. HOPEFULLY, if an employee isn't assigned a computer, the CurrentComputer type on the employee should show as null.
+
+   //by: Connor FitzGerald
+namespace BangazonAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EmployeeController : ControllerBase
+    {
+        private readonly IConfiguration _config;
+
+        public EmployeeController(IConfiguration config)
+        {
+            _config = config;
+        }
+
+        public SqlConnection Connection
+        {
+            get
+            {
+                return new SqlConnection(_config.GetConnectionString("DefaultConnection"));
+            }
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Get()
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+
+                    //joins employee, department, and computer tables
+                    string command = $@"SELECT e.Id AS 'Employee Id', e.FirstName, e.LastName, e.IsSuperVisor, e.DepartmentId,
+                        d.Id AS 'Department Id', d.Name AS 'Department', d.Budget ,c.Id AS 'Computer Id', 
+						c.Make, c.Manufacturer, c.PurchaseDate, c.DecomissionDate
+                        FROM Employee e FULL JOIN Department d ON e.DepartmentId = d.Id
+						LEFT JOIN ComputerEmployee ce ON e.Id = ce.EmployeeId
+                        LEFT JOIN Computer c ON ce.ComputerId=c.Id";
```

Comment:
This works with the way the data is currently seeded, but I added another entry to the `EmployeeComputer` table to see what happened if an employee had more than one computer in their history. It still brought back the old one, not their current one. 

For reference, I added a new entry establishing a relationship between Jacob Turner and computer number two, because his old computer (number 3) actually had an `UnassignDate` property on it. 

`INSERT  INTO ComputerEmployee (EmployeeId, ComputerId, AssignDate, UnassignDate) VALUES (2, 2, '02/13/2018', null);`

I added that entry and then made another `GET` request-- it still said Jacob's computer was number 3 (his old computer).